### DISCUSSION TITLE
There are any theoretical to explain the logic behind the function "bool HighsSearch::orbitsValidInChildNode"

### DISCUSSION THREAD

**Discussion by xtjjyygy** [ERGO-Code/HiGHS#1764] [Q&A]
![image](https://github.com/ERGO-Code/HiGHS/assets/38133593/73542c06-b4ba-4aee-8a65-fd225286f6b1)

For the second condition, if the variable is binary and a down branch, the function will return true. But if the variable is not Stabilized, which will any question?  Can you give me some theoretical documents for it? Thanks

**Comment by jajhall** [comment]
None that I know of
